models/models.js

```diff
@@ -21,4 +21,31 @@ const fetchUsers = () => {
   return db.query(`SELECT * FROM users;`).then(({ rows }) => rows);
 };
 
-module.exports = { fetchCategories, fetchReviewId, fetchUsers };
+const patchReviewVotes = (id, voteBody) => {
+  const theKey = Object.keys(voteBody);
+
+  if (theKey[0] !== "inc_votes") {
```

Comment:
can use if(!voteBody.inc_votes) in case user send extra key value pairs  e.g. {title: "banana", inc_votes: 1, newKey:"whatever"}. Checking voteBody.inc_votes is not undefined will ignore other unrelated key value pairs.

Reply:
For this error we can also use psql error handler with err.code === "23502" rather than custom error handler. But your solution is fine as well